src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -11,7 +11,18 @@ namespace System.Diagnostics
     // another version of this partial class with the public visibility 
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if MONO
```

Comment:
If our side is non public, would it make sense to just make the changes in our code instead of having #if's ? Same below.